composer.json

```diff
@@ -111,6 +111,7 @@
     "symfony/routing": "^5.3",
     "symfony/security-bundle": "^5.4.0",
     "symfony/security-core": "^5.4.0",
+    "symfony/security-guard": "^5.3",
```

Comment:
Is it intended to re-introduce the guard here?

Reply:
ops,  it's was a mistake when merging conflicts, thank you